src/plugins/github/fetchGithubRepo.js

```diff
@@ -74,7 +74,15 @@ function postQuery({body, variables}, token) {
   })
     .then((x) => x.json())
     .then((x) => {
-      if (x.errors) {
+      if (x.errors || x.data == null) {
```

Comment:
Could we check `x.data === undefined` or `!("data" in x)` instead? It is
generally possible for the result of a valid query to be exactly `null`.

Reply:
Done.